themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
To see the difference, you can play here: http://tiddlywiki.com/plugins/tiddlywiki/highlight/

![screenshot_2015-12-29_01-55-32](https://cloud.githubusercontent.com/assets/212368/12032658/60def4bc-adcf-11e5-8307-b46ed5642a5a.png)